mh/quest_utils.py

```diff
@@ -10,7 +10,19 @@
 from mh.equipment_utils import create_arena_equipment_set
 
 
-class LocationType:
+class Enum(object):
+    def __getitem__(self, idx):
+        return getattr(self, idx)
```

Comment:
This will allow us to create quests programatically from strings. For example, instead of just using the encoded number `8` to refer to barroth in a quest .json file, we can instead use the string `barroth` and the `Enum` nature of the `Monsters` class will allow us to get the appropriate number from that string (i.e. `Monsters['barroth']`, which resolves to `8`.)